src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/policy/ApplicantChangeOrgPolicyAboutToSubmitHandlerTest.java

```diff
@@ -33,15 +33,15 @@ void setup() {
     }
 
     @Test
-    void canHandle() {
+    void canHandleContested() {
```

Comment:
You can use the `shouldHandleAllCaseTypes()` to test multiple case type handlers, see `DocumentRemovalAboutToStartHandlerTest.java` for example.